sdk/key_vault/src/key.rs

```diff
@@ -0,0 +1,334 @@
+use std::fmt::{Debug, Display};
+
+use azure_core::TokenCredential;
+use chrono::serde::ts_seconds_option;
+use chrono::{DateTime, Utc};
+use getset::Getters;
+use serde::Deserialize;
+use serde_json::{Map, Value};
+use url::Url;
+
+use crate::client::API_VERSION;
+use crate::{KeyVaultClient, KeyVaultError};
+
+/// A KeyBundle consisting of a WebKey plus its attributes.
+#[derive(Debug, Deserialize, Getters)]
+#[getset(get = "pub")]
+pub struct KeyBundle {
+    /// The key management attributes.
+    attributes: KeyAttributes,
```

Comment:
`KeyAttributes` should be internal to the lib and wrapped by `KeyProperties`, which you can see here in Java: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/KeyProperties.java#L29